components/composited/SignUpForm/SignUpForm.tsx

```diff
@@ -82,7 +83,7 @@ export const SignUpForm = ({ onSubmit }: SignUpFormProps) => {
             label="Hasło (minimalnie 9 znaków)"
             minLength={8}
           />
-          <Input type="password" name="passwordConfirmation" label="Powtórz hasło" minLength={9} />
+          <Input type="password" name="passwordConfirmation" label="Powtórz hasło" minLength={8} />
```

Comment:
Actually, it's a good catch. Can you make a constant `MIN_PASSWORD_LENGTH` and pass it throughout our forms?
```suggestion
          <Input type="password" name="passwordConfirmation" label="Powtórz hasło" minLength={8} />
```